### ISSUE TITLE
which Label Smoothing is better? : tf's categoricalcrossentropy vs custom function

### ISSUE THREAD

**Issue by MokkeMeguru** [strutive07/transformer-tensorflow2.0#13]
Hello, I found label smoothing can lead us better performance for the transformer.

But I wonder why you implement label smoothing despite there is the argument of label smoothing in tf.keras.losses.CategoricalCrossEntropy (https://www.tensorflow.org/api_docs/python/tf/keras/losses/CategoricalCrossentropy?version=nightly)

So, I want to get your answer to this question.
Why you implement the custom function for label smoothing?
